modules/serverless-kafka-broker-sasl-default-config.adoc

```diff
@@ -0,0 +1,98 @@
+// Module is included in the following assemblies:
+//
+// * serverless/knative_eventing/serverless-kafka.adoc
+
+[id="serverless-kafka-broker-sasl-default-config_{context}"]
+= Configuring SASL authentication for the Kafka Broker
```

Comment:
This one has the exactly same title as https://github.com/openshift/openshift-docs/pull/39408/files#diff-741342931b1ca442259b80ba2cf945e18bebb1ff9c2c3535e6067ac10c399eb3R6
Perhaps we should differentiate them? Is this current section related to "default" configuration?

Reply:
Also, use "broker" not "Broker" in the titles